.github/workflows/pr-builder.yml

```
@@ -114,6 +114,40 @@ jobs:
       - name: Run Integration Tests
         run: make integration-test
 
+  sqlite-test:
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)